core/src/main/java/entities/POI.java

```diff
@@ -0,0 +1,57 @@
+package entities;
+
+
+
+public class POI {
+
+
+	private long id;
+	
+	private String name;
+
+	private double X;
+	
```

Comment:
the should be at least type of POI - like 'education', 'medicine' etc
`private final String type`